Gopkg.toml

```diff
@@ -71,6 +71,10 @@ required = [
   name = "k8s.io/apiserver"
   version = "kubernetes-1.11.2"
 
+[[override]]
+  name = "github.com/kubeflow/tf-operator"
```

Comment:
Why should we have such a dep?

Reply:
I want to sync the latest change of tf-operator. If I do not add this, the vendor does not change and it always use the tf-operator v0.5.0

Reply:
Gotcha.

Reply:
I think it is better to use a stable tf-operator version rather than "master" branch because the update of mxnet-operator is a little slower than tf-operator.

Reply:
I think we could use stable when tf-operator v1 released.

Reply:
@gaocegege Yes, agree with that.